twiggy/tests/all/expectations/dominators_summarizes_unreachable_items

```diff
@@ -0,0 +1,6 @@
+ Retained Bytes │ Retained % │ Dominator Tree
+────────────────┼────────────┼────────────────────────────
+             58 ┊     29.44% ┊ "function names" subsection
+             48 ┊     24.37% ┊ "local names" subsection
+             21 ┊     10.66% ┊ export "add"
+             43 ┊     21.83% ┊ [9 Unreachable Items]
```

Comment:
This is the kind of output we'll see now in text format :slightly_smiling_face: I could add a 'Total' row, but didn't want this PR to become too large.